scripts/gceworker.sh

```diff
@@ -99,7 +102,15 @@ case "${cmd}" in
     start_and_wait "${NAME}"
 
     gcloud compute scp --recurse "build/bootstrap" "${NAME}:bootstrap"
-    gcloud compute ssh "${NAME}" --ssh-flag="-A" --command="./bootstrap/bootstrap-debian.sh"
+    gcloud compute ssh "${NAME}" --ssh-flag="-A" --command="DISABLE_COCKROACHDB=$DISABLE_COCKROACHDB DISABLE_MANAGED_SERVICE=$DISABLE_MANAGED_SERVICE ./bootstrap/bootstrap-debian.sh"
+
+    # Copy the ssh keys to the gceworker, this is needed to clone the managed-service repo
+    echo -n "Enter the path to the ssh key [default ~/.ssh]: "
+    read SSH_KEY_PATH
+    if [ -z "$SSH_KEY_PATH" ]; then
+      SSH_KEY_PATH="$HOME/.ssh"
+    fi
+    gcloud compute scp --recurse "${SSH_KEY_PATH}" "${NAME}:~/.ssh"
```

Comment:
This looks like it will grab _all_ ssh keys, not just one. What does it grab?

If it grabs all the keys, that is way more than should be pushed up. 

How about making the default explicitly be the common default key of `${HOME}/.ssh/id_rsa`?